presentation/src/main/java/org/sagebionetworks/research/presentation/DisplayString.java

```diff
@@ -32,20 +32,72 @@
 
 package org.sagebionetworks.research.presentation;
 
+import android.os.Parcel;
+import android.os.Parcelable;
 import android.support.annotation.Nullable;
 import android.support.annotation.StringRes;
 
-public class DisplayString {
+import com.google.common.base.Objects;
+
+
+public class DisplayString implements Parcelable {
```

Comment:
Is there a reason we're not using @AutoValue for the parcelable? We should write unit tests if we're implementing it ourselves

Reply:
I didn't implement it I just used AndroidStudio code generation to do it, however I agree I will switch it to an autovalue class